aperturedb/transformers/common_properties.py

```diff
@@ -0,0 +1,42 @@
+from aperturedb.Subscriptable import Subscriptable
+from aperturedb.transformers.transformer import Transformer
+import traceback
+
+
+class CommonProperties(Transformer):
+    """
+    This applies some common properties to the data.
+    """
+
+    def __init__(self, data: Subscriptable, **kwargs) -> None:
+        """
+        Args:
+            data: Subscriptable object
+            adb_data_source: Data source for the data
```

Comment:
Another thing I ran into just now. How about indexing for some of these, especially data_source? How do we keep it optional?

Reply:
Check if index exists, and create if not ? Would that be OK. It's in line with DataCSVs which create indexes for properties with constraint properties.

Reply:
That sounds good. ADd it here ?

Reply:
done